os/drivers/usbhost/usbhost_enumerate.c

```diff
@@ -565,11 +577,15 @@ int usbhost_enumerate(FAR struct usbhost_hubport_s *hport, FAR struct usbhost_cl
 	/* Release temporary buffers in any event */
 
 	if (buffer != NULL) {
-		DRVR_FREE(hport->drvr, buffer);
+		if (isiobuff) {
+			DRVR_IOFREE(hport->drvr, buffer);
```

Comment:
same question as above.
If we have to maintain one of allocated memory, code in here is correct.
Otherwise we have to call DRVR_FREE before we alloc IO memory.

Reply:
DRVR_ALLOC  is for fixed memory size and DRVR_IOALLOC will allocate for requested size. These API's are board specific and hence have freed using corresponding free request.